dia/diameter_travelping.dia

```diff
@@ -6,15 +6,37 @@
 ;; 2 of the License, or (at your option) any later version.
 
 ;; It contains custom Travelping AVPs related to NAT-binding
+;; When adding AVPs make sure to use values not alrady used by the
+;; Travelping RADIUS dictionary:
+;;   https://github.com/travelping/eradius/blob/master/priv/dictionaries/dictionary.travelping
```

Comment:
Maybe make sense to use latest tag of eradius instead of using master  link

Reply:
Ok